packages/forms-web-app/src/dynamic-forms/journey-factory.js

```diff
@@ -29,7 +33,8 @@ const JOURNEY_PARAMS = {
 	[JOURNEY_TYPES.S78_QUESTIONNAIRE]: buildS78LPAJourneyParams,
 	[JOURNEY_TYPES.HAS_APPEAL_FORM]: buildHASAppellantJourneyParams,
 	[JOURNEY_TYPES.S78_APPEAL_FORM]: buildS78AppellantJourneyParams,
-	[JOURNEY_TYPES.S78_LPA_STATEMENT]: S78LpaStatementJourneyParams
+	[JOURNEY_TYPES.S78_LPA_STATEMENT]: S78LpaStatementJourneyParams,
```

Comment:
should this be called buildS78LpaStatementJourneyParams

Reply:
Updated